services/tenant-ui/frontend/src/components/profile/issuance/Endorser.vue

```diff
@@ -1,12 +1,91 @@
 <template>
-  <div v-if="canConnectEndorser || hasEndorserConn" class="my-1">
-    <p class="my-1">{{ $t('profile.connectTenantToEndorser') }}</p>
-    <InputSwitch
-      :model-value="hasEndorserConn"
-      :disabled="hasEndorserConn"
-      @change="connectToEndorser"
-    />
-
+  <div v-if="canBecomeIssuer" class="my-1">
+    <DataTable
+      v-model:filters="filter"
+      :loading="loading"
+      :value="formattedLedgers"
+      :paginator="false"
+      :rows="TABLE_OPT.ROWS_DEFAULT"
+      :rows-per-page-options="TABLE_OPT.ROWS_OPTIONS"
+      selection-mode="single"
+      data-key="ledger_id"
+      sort-field="ledger_id"
+      :sort-order="-1"
```

Comment:
Think probably best to sort alphabetically for ledgers. Would be `1` then if so.